src/base_object.h

```diff
@@ -158,8 +158,7 @@ class BaseObject : public MemoryRetainer {
 
   virtual inline void OnGCCollect();
 
-  bool is_snapshotable() const { return is_snapshotable_; }
-  void set_is_snapshotable(bool val) { is_snapshotable_ = val; }
+  virtual inline bool is_snapshotable() const { return false; }
```

Comment:
Regarding consistency: should it be `IsSnapshotable()?

Reply:
I'll let @joyeecheung make that call, she introduced this code :)

Reply:
I think our convention is similar to V8: if it's cheap then it's snake-cased, and in this case this should be pretty cheap.